src/collection/collection.helpers.tsx

```diff
@@ -475,14 +468,23 @@ export const renderCommonMetadata = (
 		className?: string
 	) => ReactNode
 ): ReactNode => {
-	const { id, lom_context, lom_classification, created_at, updated_at } = collectionOrBundle;
+	const { id, created_at, updated_at, loms } = collectionOrBundle;
+	const groupedLomsLabels = getGroupedLomsKeyValue(loms || [], 'label');
+	const isEducationLevelEnabled = enabledMetaData.includes(SearchFilter.educationLevel);
+	const isSubjectEnabled = enabledMetaData.includes(SearchFilter.subject);
+
 	return (
 		<>
-			{(!!renderEducationLevels(id, lom_context, enabledMetaData, renderSearchLink) ||
-				!!renderSubjects(id, lom_classification, enabledMetaData, renderSearchLink)) && (
+			{(isEducationLevelEnabled || isSubjectEnabled) && (
 				<Column size="3-3">
-					{renderEducationLevels(id, lom_context, enabledMetaData, renderSearchLink)}
-					{renderSubjects(id, lom_classification, enabledMetaData, renderSearchLink)}
+					{isEducationLevelEnabled &&
+						renderEducationLevels(
+							id,
+							groupedLomsLabels.educationLevel,
+							renderSearchLink
+						)}
```

Comment:
shouldn't we render educationLevel and education degrees as separate sections?
https://meemoo.atlassian.net/browse/AVO-2273

```
onderwijsniveau
label + aanklikbare niveau(s)

onderwijsgraad 
label + aanklikbare niveau(s)

vakken
label + aanklikbare niveau(s)

thema’s
label + aanklikbare niveau(s)

aangemaakt op
mag in een aparte kolom blijven, of te bekijken wat beter past

laatst aangepast op
mag in een aparte kolom blijven, of te bekijken wat beter past
```